### DISCUSSION TITLE
Laravel Nova roadmap

### DISCUSSION THREAD

**Discussion by bohdan-shulha** [laravel/nova-issues#6330] [General]
Hey,

I have started developing my new admin panel with Laravel Nova and discovered some pieces that are missing out-of-the-box.

Is there any kind of roadmap? I see that there were no new, impactful features for more than half a year and I'm getting the feeling that the project is somewhat abandoned. Also, questions are not being answered here, so this is just another red flag.

Starting to feel a little bit scammed. -_-

**Comment by webard** [comment]
Laravel Nova is in very stable state. I develop with my team quite big modular monolith app with about 200 resources, using Laravel Octane and couple customized tools and we didn't encounter any unresolvable fixes/features.

it would be nice to know in what direction Nova will be developed in version 5, but like I said, version 4 is very stable. As I know only one feature: Repeater fields is in beta state: https://nova.laravel.com/docs/resources/repeater-fields.html

I can certainly recommend Nova.

**Reply by bohdan-shulha** [reply]
Thanks for your reply. 

However, I don't say it is not stable - I haven't met any issues so far while implementing ~10 pages with different layouts.

I say that I lack features. :)

For example, adding custom buttons to the create/cancel toolbar at the bottom of the edit page. Or replacing them completely. Having easy to spin up tabs. More layout options for forms. And some other tiny improvements here and there. And, of course, getting support, which is kinda expected from a paid product.

**Reply by jeremynikolic** [reply]
You'll likely be pleased with Nova 5 which will bring improvements and new features regarding layout 😉

**Comment by orlyapps** [comment]
We have also realised many large projects with laravel nova and are super happy. Many things can be easily expanded.

But we would also be interested in a roadmap. Where is the journey going? What features are planned?

**Comment by marcobax** [comment]
Did you check out Nova Packages to see if there is a readily available plugin for your use case?
[https://novapackages.com/](https://novapackages.com/)

Second option would be to create a (resource) tool and extend the functionality to your liking, see: [https://nova.laravel.com/docs/customization/tools.html](https://nova.laravel.com/docs/customization/tools.html)

**Comment by jeremynikolic** [comment]
Currently we are in the final phase of Nova's next major version. 
We are very thankful for your feedback and will take note to improve on our communication regarding future features and releases 👍

**Reply by mariusghitoiu** [reply]
@jeremynikolic Should we expect the new major version this year?

**Reply by jeremynikolic** [reply]
We would like to but are currently holding back a bit so we may release with inertia 2 out of the box 👌

**Reply by muhammadsaeedparacha** [reply]
@jeremynikolic 
Oh wow, I thought Nova was dead and was thinking of migrating over to Filament, even though I hated moving away from VILT because our enterprise application really needs the smooth SPA .

I couldn't get any feedback on Nova roadmap, is there a public roadmap I can view for Nova? or any big features planned? The lack of visibility into this made us question if Nova was still a priority

**Reply by jeremynikolic** [reply]
@muhammadsaeedparacha we unfortunately don't have a set roadmap at this point in time

But I'll see if we may provide insights to the list of changes coming with version 5 as well as pieces that we know we want to do next 👀
